test/fixtures/api/schema-body.json

```
@@ -130,15 +130,15 @@
                                   "content": [
                                     {
                                       "element": "string",
-                                      "content": "messageBody"
+                                      "content": "messageBodySchema"
```

I don't think this is correct, the response is itself a JSON Schema with content-type (application/schema+json ; hello=world). I-e, the API is expected to return a JSON Schema in the body.

```suggestion
                                      "content": "messageBody"
```